src/font.ts

```diff
@@ -296,47 +296,39 @@ export class Font {
   };
 
   /**
-   * @param fontName
-   * @param woffURL The absolute or relative URL to the woff file.
-   * @param includeWoff2 If true, we assume that a woff2 file is in
-   * the same folder as the woff file, and will append a `2` to the url.
-   */
-  // Support distributions of the typescript compiler that do not yet include the FontFace API declarations.
-  // eslint-disable-next-line
-  // @ts-ignore
-  static async loadWebFont(fontName: string, woffURL: string): Promise<FontFace> {
-    const fontFace = new FontFace(fontName, `url(${woffURL})`);
-    await fontFace.load();
-    // eslint-disable-next-line
-    // @ts-ignore
-    document.fonts.add(fontFace);
-    return fontFace;
-  }
-
-  /**
-   * Load the web fonts that are used by your app.
-   * If fontNames is undefined, all fonts in Font.WEB_FONT_FILES will be loaded.
+   * This method is asynchronous, so you should use await or .then() to wait for the fonts to load before proceeding.
    *
-   * For example, `flow.html` calls:
-   *   `await Vex.Flow.Font.loadWebFonts();`
-   * Alternatively, you may load web fonts with a stylesheet link (e.g., from Google Fonts),
-   * and a @font-face { font-family: ... } rule in your CSS.
-   *
-   * You can customize `Font.WEB_FONT_HOST` and `Font.WEB_FONT_FILES` to load different fonts
-   * for your app.
+   * @param fontName
+   * @param url The absolute or relative URL to the woff2/otf file. It can also be a data URI.
    */
-  static async loadWebFonts(fontNames?: string[]): Promise<void> {
-    const allFiles = Font.WEB_FONT_FILES;
-    if (!fontNames) {
-      fontNames = Object.keys(allFiles);
+  static async load(fontName: string, url?: string, descriptors?: Record<string, string>): Promise<FontFace> {
```

Comment:
`Font.load()` loads a font file from a URL, or if the URL is not specified, it will load the font from the jsDelivr CDN.